site/operational-guide/tasks.md

```diff
@@ -30,22 +41,22 @@ may not be successful, or indeed possible:
 
 In some cases you may already be aware of the problem, as you were
 unable to execute `weave reset` successfully or because you know
-through other channels that the host has died - in these cases you can
-proceed straight to the section on reclaiming lost space.
+through other channels that the host has died. In these cases you can
+proceed straight to the reclaiming lost address space section.
 
 However in some scenarios it may not be obvious that space has been
 lost, in which case you can check for it periodically with the
 following command on any peer:
 
     weave status ipam
 
-This will list the names of unreachable peers; if you are satisifed
+This command lists the names of unreachable peers. If you are satisifed
```

Comment:
Hmm. The command does more than that. "Part of the output identifies unreachable peers"?